torch/distributed/rpc/_testing/__init__.py

```diff
@@ -1,6 +1,5 @@
-
 import torch
-
+from torch.distributed.distributed_c10d import ProcessGroup
```

Comment:
Why does it needs to be imported here?

Reply:
When I run testing, this symbol is used by some generated unit tests. Not importing it will cause those tests to fail. Sorry I didn't take notes of the exact tests that failed, but it should be "test/distributed/rpc/test_process_group_agent.py". Will double check.

Reply:
I double checked. Removing this line will not cause test failures, so in the latest commit I just remove it.